packages/spacecat-shared-gpt-client/src/clients/firefall-client.js

```diff
@@ -51,6 +51,7 @@ export default class FirefallClient {
     const {
       FIREFALL_API_ENDPOINT: apiEndpoint,
       IMS_CLIENT_ID: imsOrg,
```

Comment:
Can this be changed to `imsClientId` to be less confusing?

Reply:
Um.  Ya.  That would be more accurate.
Would I change line 73 too?
Feels like I'm starting to step on existing code but...